src/app/Topology/Shared/Entity/EntityDetails.tsx

```diff
@@ -214,24 +229,110 @@ export const TargetDetails: React.FC<{
     ];
   }, [serviceRef]);
 
+  const onToggle = React.useCallback(() => setExpanded((v) => !v), [setExpanded]);
+
   return (
     <DescriptionList {...props} columnModifier={columnModifier}>
-      {_transformedData.map((d) => (
-        <DescriptionListGroup key={d.key}>
-          <DescriptionListTermHelpText>
-            <Popover headerContent={d.helperTitle} bodyContent={d.helperDescription}>
-              <DescriptionListTermHelpTextButton>{d.title}</DescriptionListTermHelpTextButton>
-            </Popover>
-          </DescriptionListTermHelpText>
-          <DescriptionListDescription style={{ userSelect: 'text', cursor: 'text' }}>
-            {d.content}
-          </DescriptionListDescription>
-        </DescriptionListGroup>
-      ))}
+      {_transformedData.map(mapSection)}
+      <ExpandableSection
```

Comment:
This section should be outside the part `<DescriptionList {...props} columnModifier={columnModifier}>` since it will create a nested description list?